tests/unit/twitter.js

```diff
@@ -4,13 +4,14 @@
   var deps = [
     'assert',
     'nock',
+    'underscore',
```

Comment:
There is a lodash in package.js here https://github.com/botwillacceptanything/botwillacceptanything/blob/3142bc62204b88970b03e23db6db4fc8b7ea4df0/package.json#L21

But no underscore, making the ci-build fail.